src/cuda_packet.cpp

```diff
@@ -15,6 +15,10 @@
 #include "op.h"
 #include "log.h"
 
+static const char *reduce_op_name[(int) ReduceOp::Count] = {
```

Comment:
Would it be possible to not re-define this? (vs `cuda_scatter.cpp`).
Having it duplicated in separate locations makes it error-prone when the enum changes.
Although I see it's already defined another time in `llvm_scatter.cpp`, so maybe it's not easy to have it only once.